inspec/a2-api-integration/files/fixtures/converge/converge-failure-report_with_ignored_failure_run_converge.json

```diff
@@ -3457,7 +3457,7 @@
         "cookbook_version":"0.1.1"
      }
   ],
-  "run_id":"ba6acb91-1eaa-4c84-8d68-f19ee641e606",
```

Comment:
This is the same as this example run https://github.com/chef/automate/blob/1ca3deb611fa1ab5bbf28cfb55179011e16d4a3e/inspec/a2-api-integration/files/fixtures/converge/converge-failure-report_run_converge.json#L3426